docs/content/guides/dagster/recommended_project_structure.mdx

```diff
@@ -0,0 +1,202 @@
+- Scanning a data warehouse for tables that haven't been used in months and delete them
+
+In these cases, we recommend keeping all ops together in an 'ops_graphs/' directory. Similarly to assets, a structure under 'ops_graphs/' can be used to group jobs and their respective graphs into distinct business domains.  
+
+
+### For multiple repositories
+
+So far, we've discussed our recommendations for structuring a large project which contains only one repository. Dagster also allows you to structure a project with multiple Dagster repositories. We don't recommend over-abstracting too early, and in most cases, one repository should be sufficient. The pattern we found useful is to use multiple repositories is to keep conflicting dependencies separate, where each Dagster repository can keep their own package requirements (e.g., `setup.py`) and deployment specs (e.g., `Dockerfile`).
+
+### For other folders
+
+#### For tests
+
+Dagster believes that testing is a fundamental part of building a maintainable data application and have integrated CI/CD processes. We recommend setting up a separate folder structure that mirrors the main project, (e.g. having a folder for test assets with any applicable subfolders) which contains the unit tests for each of the components of the data pipeline. 
+- Each of the components in Dagster such as assets, sensors and resources can all be tested separately. See more on [Testing in Dagster](https://docs.dagster.io/concepts/testing)
+- For details on [Unit Tests with stubs and mocks](/guides/dagster/transitioning-data-pipelines-from-development-to-production#advanced-unit-tests-with-stubs-and-mocks) 
+
+
+```
+project_fully_featured
+├── project_fully_featured
+├── project_fully_featured_tests
+│   ├── __init__.py
+│   ├── test_assets
+│   │   └── test_recommender
+│   │       ├── test_comment_stories.py
+│   │       ├── test_user_story_matrix.py
+│   │       └── test_user_top_recommended_stories.py
+│   │   └── test_core.py
+│   ├── test_resources
+│   │   ├── test_parquet_io_manager.py
+│   │   └── test_snowflake_io_manager.py
+│   └── test_sensors
+│       ├── __init__.py
+│       ├── test_hn_tables_updated_sensor.py
+│       └── test_slack_on_failure_sensor.py
+```
+
+
+#### For other projects outside Dagster
+
+```
+project_fully_featured
+├── dbt_project
+│  ├── README.md
+│  ├── analysis
+│  ├── config
+│  │  └── profiles.yml
+│  ├── data
+│  │  └── full_sample.csv
+│  ├── dbt_project.yml
+│  ├── macros
+│  │  ├── aggregate_actions.sql
+│  │  └── generate_schema_name.sql
+│  ├── models
+│  │  ├── activity_analytics
+│  │  │  ├── activity_daily_stats.sql
+│  │  │  ├── comment_daily_stats.sql
+│  │  │  └── story_daily_stats.sql
+│  │  ├── schema.yml
+│  │  └── sources.yml
+│  ├── snapshots
+│  ├── target
+│  │  └── manifest.json
+│  └── tests
+│      └── assert_true.sql
+├── project_fully_featured
+│ ...
+```
+
+## Other recommendations 
```

Comment:
```suggestion
---

## Other recommendations 
```

🔧 H2 headings should be preceded by a horizontal rule; it helps break things up visually